src/main/scala/mesosphere/marathon/Main.scala

```diff
@@ -61,11 +62,17 @@ class MarathonApp(args: Seq[String]) extends AutoCloseable with StrictLogging {
         """.stripMargin
       }
       val statsd = cliConf.graphite.get.map { urlStr =>
-        val url = new URI(urlStr)
+        val url = Uri(urlStr)
+
+        if (url.scheme.toLowerCase() != "udp") {
+          logger.warn(s"Graphite reporter protocol ${url.scheme} is not supported; using UDP")
+        }
+        val params = url.query()
         s"""
            |kamon.statsd {
-           |  hostname: ${url.getHost}
-           |  port: ${if (url.getPort == -1) 8125 else url.getPort}
+           |  hostname: ${url.authority.host}
+           |  port: ${if (url.authority.port == 0) 8125 else url.authority.port}
+           |  flush-interval: ${params.collectFirst { case ("interval", iv) => iv.toInt }.getOrElse(10)} seconds
```

Comment:
is it documented somewhere that 10 is a default interval?

Reply:
This default is specified in the upstream Kamon StatsD dependency

Reply:
(And our documentation already specifies that the default is 10)